benchconnect/benchconnect/_cli.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+import click
+from benchclients.logging import log
+
+import benchconnect._augment as _augment
+import benchconnect._post as _post
+import benchconnect._put as _put
+
+log.setLevel(logging.DEBUG)
+
+
+@click.group(help="Command line utilities for interacting with a Conbench API")
+def cli():
+    pass
+
+
+@cli.group(help="Fill in missing fields of a JSON blob with defaults")
+def augment():
+    pass
+
+
+augment.add_command(_augment.result, name="result")
+augment.add_command(_augment.run, name="run")
+
+
+@cli.group(
+    help="""
+Post something to a Conbench API
+
+\b
```

Comment:
huh TIL about the backspace character. What is it doing here?

Reply:
It's a click thing for preserving newlines: https://stackoverflow.com/a/53302580/4497050 I looked a bit for a better way to get `<br>` instead of `<p>`, but this seems like all there is